tigerpath/majors_and_certificates/majors/MUS_2018.json

```
@@ -0,0 +1,165 @@
+{
+  "type": "Major",
+  "name": "Music",
+  "code": "MUS",
+  "degree": "AB",
+  "year": 2018,
+  "explanation": "The Department of Music aims to give students a broad foundation for making, performing, studying, and writing about music. As part of the major, students pursue work in two loosely defined areas: Culture and Criticism (C&C) and Materials and Making (M&M). M&M encompasses topics usually explored in hands-on, exploratory ways, such as composition, improvisation, theory, analysis, and electronic music. The C&C area focuses on the scholarly study of music, from historical and formalist analysis to cognitive science and ethnography. Whether our concentrators ultimately choose to make music, to research and write about it, or do both, they are encouraged to develop independent work that moves across traditional disciplinary or methodological boundaries.",
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-music",
+    "https://music.princeton.edu/program-study"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Wendy Heller",
+      "email": "wbheller@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Juri Seo",
+      "email": "juri@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "MUS 105, MUS 106, and a 200-level Culture & Criticism course. Students who place out of MUS 105 by demonstrating equivalent knowledge in an exam given by the music faculty are required to take an additional elective to meet the minimum requirement of 11 courses in the major.",
+      "req_list": [
+        {
+          "name": "Required",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "course_list": [
+            "MUS 105",
+            "MUS 106"
+          ]
+        },
+        {
+          "name": "Culture & Criticism",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": null,
+          "course_list": [
+            "MUS 230",

```

Some courses from the list on the [department website](https://music.princeton.edu/culture-and-criticism) seem to be missing.
```suggestion
            "MUS 203",
            "MUS 220",
            "MUS 223",
            "MUS 230",
```